### PR TITLE
[RA2 Ch4]: Correction of link to README.md

### PR DESCRIPTION
**Pull request by CsatariGergely**
Fixes: #2559

### CODE REVIEW

File: doc/ref_arch/kubernetes/chapters/chapter04.md [hunk 1]
```diff
@@ -73,12 +73,12 @@ In order for the Kubernetes components to be conformant with the Reference Archi
...
+|`ra2.k8s.001`|Kubernetes Conformance|The Kubernetes distribution, product, or installer used in the implementation **must** be listed in the [Kubernetes Distributions and Platforms document](https://docs.google.com/spreadsheets/d/1uF9BoDzzisHSQemXHIKegMhuythuq_GL3N1mlUUK2h0/edit#gid=0) and marked (X) as conformant for the Kubernetes version defined in [README](../README.md#required-versions-of-most-important-components) under "Required versions of most important components".|[req.gen.cnt.03](./chapter02.md#23-kubernetes-architecture-requirements)|[4.3.1](/doc/ref_impl/cntt-ri2/chapters/chapter04.md#431-installation-on-bare-metal-infratructure)|
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter04.md:76]: @CsatariGergely the link is still "../README.md#required-versions" (Line 48, README)

Comment (CsatariGergely) [reply doc/ref_arch/kubernetes/chapters/chapter04.md:76]: It is https://github.com/cntt-n/CNTT/tree/master/doc/ref_arch/kubernetes#required-versions-of-most-important-components on GitHub and https://cntt.readthedocs.io/en/latest/ref_arch/kubernetes/README.html#required-versions-of-most-important-components on Readthedocs.

Comment (CsatariGergely) [reply doc/ref_arch/kubernetes/chapters/chapter04.md:76]: These are automatically generated, so I've deleted the anchor for "required-versions" from README.

**Comment by pgoyal01** [comment]
@rgstori @walterkozlowski ready for merge

**Pull request merged**